wd_core.au3

```diff
@@ -888,7 +888,7 @@ EndFunc   ;==>_WD_ExecuteScript
 ;                  $sOption  - [optional] a string value. Default is ""
 ; Return values .: Success - Requested data returned by web driver.
 ;                  Failure - "" (empty string) and sets @error to one of the following values:
-;                  - $_WD_ERROR_Exception
+;                  - $_WD_ERROR_NoAlert

```

Comment:
$_WD_ERROR_NoAlert --- Where this is set?

Reply:
__WD_DetectError, which gets called by __WD_Get, __WD_Post, etc.

Reply:
aa...